### ISSUE TITLE
Term Request: Beaten metal appearance of corneal endothelium

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11754]
**Preferred term label:**
Beaten metal appearance of corneal endothelium

**Synonyms**
* Beaten bronze endothelial appearance
* Beaten silver corneal endothelium
* Hammered silver appearance of corneal endothelium

**Definition (free text, please give PubMed ID)**
A biomicroscopic appearance of the posterior corneal surface characterized by an irregular, hammered or beaten metal texture, resulting from confluent excrescences (guttae) and thickening of Descemet membrane (PMID: 38359414).

**Parent term (use hpo.jax.org/app)**
Abnormal corneal endothelium morphology (HP:0011488)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
* Fuchs endothelial corneal dystrophy (FECD) (ORPHA:98974; FECD1 OMIM:136800, FECD3 OMIM:613267)

**Your nano-attribution (ORCID)**
0000-0001-9042-0508
